JSIL.Libraries/Includes/Core/Reflection/Classes/System.Type.js

```diff
@@ -328,7 +335,7 @@
           $jsilcore.TypeRef("System.Array", [$jsilcore.TypeRef("System.Reflection.ParameterModifier")])
       ], []),
       function GetConstructor(bindingAttr, binder, callConvention, types, modifiers) {
-        return getConstructorImpl(this, bindingAttr, types);
+        return getConstructorImpl(this, bindingAttr | System.Reflection.BindingFlags.DeclaredOnly, types);
```

Comment:
The bitwise or operator on enums in JSIL doesn't necessarily do what you want, since it produces an int. This is probably fine, though...